builtins/amp-img.js

```diff
@@ -123,9 +123,10 @@ export class AmpImg extends BaseElement {
     // only read "Graphic" when using only 'alt'.
     if (this.element.getAttribute('role') == 'img') {
       this.element.removeAttribute('role');
-      user().error('AMP-IMG', 'Setting role=img on amp-img elements breaks ' +
-        'screen readers please just set alt or ARIA attributes, they will ' +
-        'be correctly propagated for the underlying <img> element.');
+      user(this.element).error('AMP-IMG',
```

Comment:
@zhouyx I need that change to amp-img.js to do the test, but the whole migration to use(opt_element) would be in a separate PR. Is that ok?